infra/README.md

```diff
@@ -0,0 +1,61 @@
+Akka Jenkins Infra
+==================
+
+Dependencies
+------------
+
+```
+bundle install
+chef gem install knife-solo
+
+berks vendor
+```
+
+Then prepare the lifflechef file:
+
+```
+mv littlechef.cfg-example littlechef.cfg
+```
+
+And fill it in with potential secrets etc.
+
+Preparing a node for cheffing
+-------------
+
+```
+fix node:cassandra-01 deploy_chef
+```
+
+Adding a new akka node automatically
+------------------------------------
+
+```
+./gcloud-new-akka-node.sh akka-node-002 "[\"10.132.0.7\", \"10.132.0.8\"]"
+```
+
+Cooking nodes
+-------------
+
+Executing scripts on a given node:
+
+```
+fix node:moxie-a0 [role:specific-role]
```

Comment:
Yeah I added the new things next to the old things since some overlap. It would be best to not split it into more projects. I would be open to calling this Akka-infra-scripts or something if we want (as such repo)